src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -31,9 +31,9 @@ def import_from_package
       end
 
       def show
-        # Because the form needs a Description & PreferenceType objects instantiated
+        # Because the form needs a Description & Preference objects instantiated
         @image.build_description if @image.description.nil?
-        @image.build_preference_type(image_type: 'docker') if @image.preference_type.nil?
+        @image.build_preference(type_image: 'docker') if @image.preference.nil?
```

Comment:
[nitpick] But why do we set docker?

We missed this in the previous PR, sorry! But we should fix that in a later PR.